examples/camera_viz/scripts/_install_deps.sh

```
@@ -79,6 +82,8 @@ while (( $# )); do
         --zed-sdk)      ZED_SDK_DIR=$2; shift 2;;
         --build-from-source) BUILD_FROM_SOURCE=true; shift;;
         *) die "unknown arg: $1";;
+        --with-orbbec)  WITH_ORBBEC=true; shift;;
```

Shellcheck (0.11.0)

[warning] This pattern never matches because of a previous pattern on line 84.

(SC2222)